liquibase-standard/src/main/java/liquibase/command/core/ChangelogSyncCommandStep.java

```diff
@@ -57,7 +57,9 @@ public void run(CommandResultsBuilder resultsBuilder) throws Exception {
         final Database database = (Database) commandScope.getDependency(Database.class);
         final DatabaseChangeLog changeLog = (DatabaseChangeLog) commandScope.getDependency(DatabaseChangeLog.class);
         final ChangeLogParameters changeLogParameters = (ChangeLogParameters) commandScope.getDependency(ChangeLogParameters.class);
-
+        final ChangeLogHistoryService changeLogHistoryService = Scope.getCurrentScope().getSingleton(ChangeLogHistoryServiceFactory.class).getChangeLogService(database);
```

Comment:
Interesting.. any idea why it stopped working here as it is not related to AbstractUpdateCommandStep?

Reply:
It's because I removed the call to do this in DatabaseChangelogCommandStep, so I've got to add it where it was dependent.